service/provider/src/main/java/gov/va/vro/service/provider/mas/MasProcessingObject.java

```diff
@@ -72,4 +72,8 @@ public String getDisplayName() {
   public String getClaimSubmissionDateTime() {
     return claimPayload.getClaimDetail().getClaimSubmissionDateTime();
   }
+
+  public String getOffRampReason() {
+    return claimPayload.getOffRampReason();
```

Comment:
I think we will need to make this an array since there might be multiple of these under rare circumstances. When and error and offramp adds up or error from both mas and lighhouse. Not this PR.